modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -6,17 +6,17 @@
 [id="updating-eus-to-eus-upgrade_{context}"]
 = EUS-to-EUS update
 
-The following procedure pauses all non-master MachineConfigPools and performs updates from {product-title} 4.8 to 4.9 to 4.10, then unpauses the previously paused MachineConfigPools.
+The following procedure pauses all non-master MachineConfigPools and performs updates from {product-title} 4.10 to 4.11 to 4.12, then unpauses the previously paused MachineConfigPools.
 Following this procedure reduces the total update duration and the number of times worker nodes are restarted.
 
 .Prerequisites
 
-* Review the release notes for {product-title} 4.9 and 4.10
+* Review the release notes for {product-title} 4.11 and 4.12
 * Review the release notes and product lifecycles for any layered products and OLM Operators. Some may require updates either before or during an EUS-to-EUS update.
-* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.8 to 4.9.
-* Verify that your cluster is running {product-title} version 4.8.14 or later.
-If your cluster is running a version earlier than {product-title} 4.8.14, you must update to a later 4.8.z version before updating to 4.9.
-The update to 4.8.14 or later is necessary to fulfill the minimum version requirements that must be performed without pausing MachineConfigPools.
+* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.11/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.10 to 4.11.
+* Verify that your cluster is running {product-title} version 4.10.14 or later.
```

Comment:
This section might no longer be applicable at all  - check w/ SMEs

Reply:
@petr-muller @wking what do y'all think? :)

Reply:
removed per @wking 's comment :)